pkg/app/app_test.go

```diff
@@ -1502,7 +1502,7 @@ releases:
 		SetFilter(true),
 	)
 
-	expected := "in ./helmfile.yaml: duplicate release \"foo\" found in namespace \"foo\" in kubecontext \"default\": there were 2 releases named \"foo\" matching specified selector"
+	expected := "in ./helmfile.yaml: duplicate release \"foo\" found in namespace \"foo\" in kubecontext \"foo\": there were 2 releases named \"foo\" matching specified selector"
```

Comment:
Apparently, you found a bug in the implementation and the test case here. Good job!

Reply:
@mumoshu Thanks very much.

Reply:
Well I read code twice and this wasn't actually a bug. `OverrideKubeContext` is meant for "overriding" kubecontext that is already configured in the helmfile.yaml releases[].kubeContex. Same for `OverrideNamespace`. You made OverrideKubeContext actually a "default kubecontext" in this pull request, which isn't how the field is supposed to work.
That said, I think you need to revert changes on `ApplyOverrides`.